code/implementation/docker/machine/src/main/resources/schema/base/amazonec2Config.json.d/amazonec2.json

```diff
@@ -12,8 +12,7 @@
         },
         "vpcId": {
             "type": "string",
```

Comment:
We have to add an API filter to check if either vpcId or subnetId is specified, but not both